doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -366,12 +367,12 @@ the monitor nodes with the `iburst` option and multiple peers:
    into initial synchronization.
 
 Furthermore, it is advisable to synchronize *all* nodes in your cluster against
-internal and external servers, and perhaps even against your monitors. ``NTP``
-servers should run on bare metal; VM virtualized clocks are not suitable for
-steady timekeeping. For more information, visit `https://www.ntp.org
-<https://www.ntp.org>`_.  Your organization might already have quality internal
-``NTP`` servers available.  Sources for ``NTP`` server appliances include the
-following:
+internal and external servers, and perhaps even against your monitors. Run
+``NTP`` servers should on bare metal; VM-virtualized clocks are not suitable
+for steady timekeeping. See `https://www.ntp.org <https://www.ntp.org>`_ for
+more information about the Network Time Protocol (NTP) Project. Your
```

Comment:
s/Project//

Reply:
This is actually what it's called on the page, but since your fresh eyes perceived it as wrong, we're going with what your eyes tell us.

Reply:
I don't doubt that it is.  My assumption was that we seek to inform the gentle reader about NTP the protocol, not the specific legacy reference implementation.  There are few reasons these days for anyone to not use the Chrony implementation.